src/uci_handler.h

```diff
@@ -369,5 +370,8 @@ class UciHandler {
         ucioption::make<ucioption::check>("Ponder", false, [](bool enabled) { s_evaluator.setPondering(enabled); }),
         ucioption::make<ucioption::string>("Syzygy", "<empty>", syzygyCallback),
         ucioption::make<ucioption::spin>("Hash", s_defaultTtHashTableSizeMb, ucioption::Limits { .min = 1, .max = 1024 }, [](uint64_t val) { engine::TtHashTable::setSizeMb(val); }),
+        ucioption::make<ucioption::spin>("Threads", 1, ucioption::Limits { .min = 1, .max = 256 }, [](uint64_t val) {
```

Comment:
I think that this should set the searchers directly. Then it's no need to store a local copy.
Also with https://github.com/hansbinderup/meltdown-chess-engine/pull/72 it's not necessary to pass to bench :)